packages/story-utils/README.md

```diff
@@ -70,3 +70,17 @@ test('my button 2', () => {
   expect(screen.getByText('coffee!')).toBeTruthy();
 });
 ```
+
+### generateSnapshots
+
+Auto generates snapshots for stories
+
+```js
+
+import * as snapshotTestStoryFile from "./MyButton.stories";
+import { generateSnapshots } from "@chanzuckerberg/story-utils";
+
+describe("My button", () => {
+  generateSnapshots(snapshotTestStoryFile);
+});
+```
```

Comment:
Just remembered this, and we can add it in a followup if you want:

We should document how to skip generating snapshot tests for a story

```js
SomeStory.parameters = {
  snapshot: {
    disabled: true,
  },
};
```

Reply:
good call out!